beaconclient/multi_beacon_client.go

```diff
@@ -212,6 +209,17 @@ func (c *MultiBeaconClient) beaconInstancesByLastResponse() []IBeaconInstance {
 	return instances
 }
 
+// beaconInstancesByLastResponse returns a list of beacon clients that has the client
+// with the last successful response as the first element of the slice
+func (c *MultiBeaconClient) beaconInstancesByLeastUsed() []IBeaconInstance {
```

Comment:
should be by last used?

Reply:
adding more comments in another PR, this is "least" used, which is "last used" in reverse order (so getting all validators is using the least used CL client)